include/eve/module/core/decorator/kind.hpp

```diff
@@ -66,4 +66,67 @@ namespace eve
   //!  @}
   //================================================================================================
   inline constexpr q_kind_type const q_kind = {};
+
```

Comment:
can `kind_1` etc be `kind<N>` ?

Reply:
yes if I decide to implement more than 2 kinds. Can do it latter/